modules/tensor_mechanics/include/materials/ADLAROMANCEStressUpdateBase.h

```diff
@@ -29,7 +29,8 @@ class ADLAROMANCEStressUpdateBase : public ADRadialReturnCreepStressUpdateBase
   virtual void initialSetup() override;
 
   virtual void initQpStatefulProperties() override;
-
+  virtual void computeStressInitialize(const ADReal & /*effective_trial_stress*/,
```

Comment:
Don't comment out the variable names in the header. You only need to do that in the .C file if they are unused.